### PR TITLE
config files for ITER loco project

### PR DESCRIPTION
**Pull request by maggia80**
The project uses an MC4-PLUS with 2 DC motors and 2 solenoids

### CODE REVIEW

File: experimentalSetups/mc4plus-loco-project/robotMotorGui.ini [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+
```
Comment (pattacini) [inline experimentalSetups/mc4plus-loco-project/robotMotorGui.ini:80]:
```suggestion
```

File: experimentalSetups/mc4plus-loco-project/yarpmotorgui.ini [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+ 
+
```
Comment (pattacini) [inline experimentalSetups/mc4plus-loco-project/yarpmotorgui.ini:79]:
```suggestion
 ```

**Comment by Nicogene** [comment]
Hi @maggia80 , what is the status of this PR? Is it ready to be merged?
I am checking the open PR since on 31/05 we have the Distro release

**Comment by pattacini** [comment]
> Hi @maggia80 , what is the status of this PR? Is it ready to be merged?
> I am checking the open PR since on 31/05 we have the Distro release

This PR is not blocking

**Review by davidetome — APPROVED** [review]

**Comment by davidetome** [comment]
to be checked on a real setup

**Pull request merged**